easybuild/easyconfigs/t/TAU/TAU-2.22.2-gompi-1.4.12-no-OFED.eb

```diff
@@ -0,0 +1,34 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# License::   New BSD
+#
+# This work is based from experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+name = 'TAU'
+version = '2.22.2'
+
+homepage = 'http://tau.uoregon.edu'
+description = """The TAU Performance System is a portable profiling and tracing toolkit
+ for performance analysis of parallel programs written in Fortran, C, C++, Java, Python."""
+
+toolchain = {'name': 'gompi', 'version': '1.4.12-no-OFED'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.cs.uoregon.edu/research/paracomp/tau/tauprofile/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('OTF', '1.12.4'),
+    ('PAPI', '5.2.0'),
+    ('PDT', '3.19'),
+    ('Scalasca', '1.4.3'),
+    ('Score-P', '1.2.1'),
```

Comment:
@geimer I'm running into a version conflict for Cube because of these two deps.

Scalasca 1.x is required for Tau, and this particular version depends on Cube 3.4.3, while Score-P 1.2.1 depends on Cube 4.2.

What's the best course of action to resolve this? Scalasca 1.x with Cube 4.2 as dep, or a variant of Score-P 1.2.1 that depends on Cube 3.4.3?